test/formatters/apheleia-ft.el

```diff
@@ -292,30 +294,36 @@ environment variable, defaulting to all formatters."
         ;; Some formatters use the current file-name or buffer-name to interpret the
         ;; type of file that is being formatted. Some may not be able to determine
         ;; this from the contents of the file so we set this to force it.
-        (rename-buffer in-file)
+        (rename-buffer (file-name-nondirectory in-file))
```

Comment:
Use `file-name-nondirectory` would have made the behavior you saw less confusing - it's unusual to set buffer names to full filesystem paths. Normally they are just basenames.